manifests/client/config.pp

```diff
@@ -133,6 +133,11 @@
     'absent' => 'rm TLS_REQCERT',
     default  => "set TLS_REQCERT ${::openldap::client::tls_reqcert}",
   }
+  $tls_moznss_compatibility = $::openldap::client::tls_moznss_compatibility ? {
+    undef    => undef,
+    'absent' => 'rm TLS_MOZNSS_COMPATIBILITY',
```

Comment:
`absent` is not an allowed value for that Enum.

Reply:
Ugh.  Good catch, thanks.  Added in the next commit, hopefully.